sunshine/src/main/java/io/github/tatools/sunshine/JunitSuite.java

```diff
@@ -1,23 +1,25 @@
 package io.github.tatools.sunshine;
 
+import java.util.List;
+
 /**
  * @author Dmytro Serdiuk (dmytro.serdiuk@gmail.com)
  * @since 06.04.2017
  */
 public class JunitSuite implements Suite<Class<?>[]> {
 
-    private final TestClasses testClasses;
+    private final Suite<List<Test<Class>>> classesAsSuite;
 
-    public JunitSuite(Location location) {
```

Comment:
Maybe, which filter we have to use then? Initial idea was that user provides a list of classes or a location which has to be filtered before run. Just submit new ticket if you like to see it with appropriate comments.